pyproject.toml

```diff
@@ -48,9 +48,6 @@ empack = "empack.cli.main:app"
 [project.urls]
 Homepage = "https://github.com/emscripten-forge/empack"
 
-[tool.hatch.build.targets.wheel.shared-data]
-config = "share/empack"
```

Comment:
I hope I won't be too annoying to you on this. I have a preference for this to stay 🫤 

I believe we should make sure we can find this path when using pyenv.

It seems jupyter has something like that https://github.com/jupyter/jupyter_core/blob/main/jupyter_core/paths.py#L102 for finding whether or not it should use user-level paths or conda environment paths.

Reply:
> I hope I won't be too annoying to you on this.

Absolutely not! :D I'd also love to ensure `pyenv` works and not move anything else. Let me port Jupyter's thing here, then.